jepsen/src/jepsen/net.clj

```diff
@@ -46,75 +48,118 @@
 
 (def tc "/sbin/tc")
 
+(defn net-dev
```

Comment:
OH hey this is great--I was *just* hitting issues with some other Jepsen code that assumed "eth0" as the primary iface. Thanks for this!

Reply:
The function is from the net-flaky-nemesis.clj gist you provided. :)

Reply:
Ha! That tells you something about how flaky my memory is, lol ;-)